content/en/integrations/vmware_tanzu_application_service.md

```diff
@@ -122,52 +122,49 @@ cf set-env <YOUR_APP> DD_TAGS key1=value1,key2=value2
 
 ## Data Collected
 
 ### Metrics
 
-The following metrics are sent by the Datadog Firehose Nozzle and are prefixed with `cloudfoundry.nozzle`. The Datadog Agent sends metrics from any Agent checks you configure in the Director runtime configuration, and [system][24], [network][25], [disk][26], and [NTP][27] metrics by default.
+The following metrics are sent by the Datadog Firehose Nozzle and are prefixed with `cloudfoundry.nozzle`. The Datadog Agent sends metrics from any Agent checks you configure in the Director runtime configuration, and [system][15], [network][16], [disk][17], and [NTP][18] metrics by default.
 
 The Datadog Firehose Nozzle only collects CounterEvents (as metrics, not events), ValueMetrics, and ContainerMetrics; it ignores LogMessages and Errors.
 
-Your specific list of metrics may vary based on the PCF version and the deployment. Datadog collects counter and gauge metrics emitted from the [Loggregator v2 API][28]. See [Cloud Foundry Component Metrics][29] for a list of metrics emitted by default.
+Your specific list of metrics may vary based on the PCF version and the deployment. Datadog collects counter and gauge metrics emitted from the [Loggregator v2 API][19]. See [Cloud Foundry Component Metrics][20] for a list of metrics emitted by default.
 
 {{< get-metrics-from-git "cloud_foundry">}}
 
 ## Further reading
 
 {{< partial name="whats-next/whats-next.html" >}}
 
-[1]: https://network.pivotal.io/products/datadog
-[2]: https://network.pivotal.io/products/datadog-application-monitoring
-[3]: https://tanzu.vmware.com/pivotal#:~:text=Pivotal%20Cloud%20Foundry%20(PCF)%20is%20now%20VMware%20Tanzu%20Application%20Service.
-[4]: https://docs.cloudfoundry.org/buildpacks/understand-buildpacks.html#supply-script
-[5]: /metrics/custom_metrics/dogstatsd_metrics_submission/
-[6]: /integrations/guide/pivotal-cloud-foundry-manual-setup
-[7]: https://docs.pivotal.io/partners/datadog-application-monitoring/installing.html
-[9]: https://docs.pivotal.io/partners/datadog/installing.html
-[10]: /agent/logs/proxy/
-[11]: /integrations/guide/pivotal-cloud-foundry-manual-setup#monitor-your-applications
-[12]: /tracing/setup/
-[13]: /integrations/guide/pivotal-cloud-foundry-manual-setup#monitor-your-pivotal-cloud-foundry-cluster
-[14]: /libraries/
-[24]: /integrations/system/#metrics
-[25]: /integrations/network/#metrics
-[26]: /integrations/disk/#metrics
-[27]: /integrations/ntp/#metrics
-[28]: https://github.com/cloudfoundry/loggregator-api
-[29]: https://docs.cloudfoundry.org/running/all_metrics.html
-[30]: /profiler/enabling/
-[32]: /integrations/faq/pivotal_architecture
-[33]: /developers/dogstatsd/
-[34]: /events/explorer/
\ No newline at end of file
+[1]: https://tanzu.vmware.com/pivotal#:~:text=Pivotal%20Cloud%20Foundry%20%28PCF%29%20is%20now%20VMware%20Tanzu%20Application%20Service
```

Comment:
manually url-encoded `(` and `)` in (PCF) here: `https://tanzu.vmware.com/pivotal#:~:text=Pivotal%20Cloud%20Foundry%20(PCF)%20is%20now%20VMware%20Tanzu%20Application%20Service.`

not encoding here makes the format-links pre-commit cut off the link